src/main/java/com/frameworkium/core/common/listeners/CreateJiraCycleListener.java

```diff
@@ -0,0 +1,181 @@
+package com.frameworkium.core.common.listeners;
+
+import com.beust.jcommander.ParameterException;
+import com.frameworkium.core.common.properties.Property;
+import com.frameworkium.core.common.reporting.jira.dto.cycle.*;
+import com.frameworkium.core.common.reporting.jira.dto.execution.AddTestToCycleOperationDto;
+import com.frameworkium.core.common.reporting.jira.dto.project.ProjectDto;
+import com.frameworkium.core.common.reporting.jira.dto.version.VersionDto;
+import com.frameworkium.core.common.reporting.jira.service.JiraService;
+import io.qameta.allure.TmsLink;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.NotFoundException;
+import org.testng.*;
+import org.testng.internal.ConstructorOrMethod;
+
+import java.util.*;
+import java.util.Map.Entry;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static java.text.MessageFormat.format;
+
+public class CreateJiraCycleListener implements ISuiteListener {
+    private final Logger logger = LogManager.getLogger(CreateJiraCycleListener.class);
+    private final JiraService jiraService = new JiraService();
+
+    @Override
+    public void onStart(ISuite suite) {
+        if (!hasResultVersionSpecified() || !hasZapiRegexCycleSpecified()) {
+            logger.info("ResultVersion and/or ZapiCycleRegex not set. "
+                    + "Skipping CreateJiraCycleListener as it is not needed...");
```

Comment:
This could be a bit more descriptive to describe what is actually not being done because the properties are not set rather than just "Skipping CreateJiraCycleListener...". 
Suggestion: - "Skipping automated generation of Jira version and/or Zapi cycle as it is not needed..."

Reply:
made the suggested changes. Thanks